### PR TITLE
Load data from a CSV

### PR DESCRIPTION
**Pull request by imruahmed**
Until the API is setup, we're loading directly from a CSV version of the data on AirTable.
Keep in mind, this is for the time being. This code should be refactored in the future to be have better structure (on the backend).

### CODE REVIEW

**Comment by jennyzsun** [comment]
++ lgtm

@andyzg do you have any comments?

**Review by andyzg — APPROVED** [review]

File: app/assets/javascripts/components/archive/ArchiveApp.es6.jsx [hunk 1]
```diff
@@ -4,8 +4,17 @@ class ArchiveApp extends React.Component {
   }
 
   render() {
+  	var videoComponents = this.props.data.map(function(video) {
+	    return (
+	       <ArchiveVideoCard 
+	          video_id={video["IDv2"]} 
+	          video_title='Edith speaking Spanish'
```
Comment (andyzg) [inline app/assets/javascripts/components/archive/ArchiveApp.es6.jsx:11]: Any way we can generate a title from the metadata?

Comment (imruahmed) [reply app/assets/javascripts/components/archive/ArchiveApp.es6.jsx:11]: I looked into it. There's not actually a title field. For now, I think we leave it like this and have a TODO to generate the title.

Comment (andyzg) [reply app/assets/javascripts/components/archive/ArchiveApp.es6.jsx:11]: sounds good. If there's an easy way to do it, we can get this done by Friday before the call with Freddie.

**Pull request merged**